backend/config/webauthn.go

```diff
@@ -10,11 +10,12 @@ var (
 	err      error
 )
 
+// TODO make these values configurable via environment variables
 func init() {
 	wconfig := &webauthn.Config{
-		RPDisplayName: "Lynx Locks",                                               // Display Name for your site
-		RPID:          "localhost",                                                // Generally the FQDN for your site
-		RPOrigins:     []string{"http://localhost:3000", "http://localhost:5001"}, // The origin URLs allowed for WebAuthn requests
+		RPDisplayName: "Lynx Locks",                                                                             // Display Name for your site
+		RPID:          "app.lynx-locks.com",                                                                     // Generally the FQDN for your site
```

Comment:
We should load this in from the environment instead of hardcoding, i still want webauthn to work on localhost

Reply:
I know, this is in the TODO on line 13. I'll just add it to this PR.

Reply:
oh sorry i didnt see the TODO. Yeah let's add it here so that i can still use it locally without any changes

Reply:
Pushed a new commit with these changes.
The domain is now configurable in prod, but will use localhost for dev and app.lynx-locks.com for prod by default if not specified.

Reply:
awesome